src/series.jl

```diff
@@ -343,8 +343,9 @@ end
 
 @recipe function f(f::FuncOrFuncs)
```

Comment:
I was planning to change `plot(sol)` to return a function, but if `plot(f)` doesn't work, then I am sure `plot(sol)` won't work. If there's an argument to set the axis, then it would still be able to work, but if you have to create a separate plot first, then I'm not sure how to do that in a recipe.